### DISCUSSION TITLE
`<c-c><c-c>` doesn't work

### DISCUSSION THREAD

**Discussion by azzamsa** [NeogitOrg/neogit#1053] [Q&A] [answered]
Hi.

After setting up https://github.com/LunarVim/Launch.nvim/blob/master/lua/user/neogit.lua (no any changes made to the repo). ` ["<c-c><c-c>"] = "Submit",` doesn't work for me.

![image](https://github.com/NeogitOrg/neogit/assets/17734314/3e7206e6-94a0-4d9d-9937-3fc7808d0e32)

It gets `type :qa! ...` everytime I type `c-c c-c`

Thanks for Neogit! 🎖️

**Comment by CKolkey** [comment, marked as answer]
Ah, lunar vim seems to pin to commit `eb9d6b8f0840e4f2fa775bca7ec4a5df8b42ed6d` which is from a few months ago. https://github.com/LunarVim/Launch.nvim/blob/master/lua/user/neogit.lua#L4C13-L4C53

I just added this mapping a few days ago, so, thats why :)

**Reply by azzamsa** [reply]
Worked like a charm! 🎖️

**Reply by CKolkey** [reply]
Nice!
